### PR TITLE
[REG-73] [ADD] set logo and background for hotel

### PR DESCRIPTION
**Pull request by RosSinch**

### CODE REVIEW

File: addons/regency_shopsite/static/src/js/frontend/header/store.js [hunk 1]
```diff
@@ -30,12 +30,30 @@ if (hotelSelectorData) {
         }
 
         get currentHotelName() {
-            if (!this.activeHotel || !this.hotels[this.activeHotel]) {
+            if (this.isNoActiveHotel) {
```
Comment (artemShelest) [inline addons/regency_shopsite/static/src/js/frontend/header/store.js:33]: Do not use negation in boolean names. `this.isActiveHotel` is better than `!this.isNoActiveHotel`

File: addons/regency_shopsite/models/res_partner.py [hunk 2]
```diff
@@ -5,3 +5,25 @@ class ResPartner(models.Model):
...
+    background_url = fields.Text(compute='_get_background_url')
+
+    def _compute_logo_url(self):
+        for partner in self:
+            image_id = partner.id
+            image_model = partner._name
+            image_field = 'image_256'
+            partner.logo_url = f'/web/image?model={image_model}&id={image_id}&field={image_field}'
+
+    def _get_background_url(self):
```
Comment (artemShelest) [inline addons/regency_shopsite/models/res_partner.py:19]: Use uniform compute method naming (`_compute_` prefix)

**Review by artemShelest — CHANGES_REQUESTED** [review]

File: addons/regency_shopsite/static/src/js/frontend/header/store.js [hunk 3]
```diff
@@ -52,6 +70,16 @@ if (hotelSelectorData) {
...
+            this._setActiveHotelBackground()
+        }
+
+        _setActiveHotelBackground() {
+            let backgroundUrl = this.currentHotelBackgroundUrl
+            let cssRec = 'background: ""'
+            if (backgroundUrl && !this.isNoActiveHotel) {
+                cssRec = `background: url(${backgroundUrl}) center center no-repeat;`
+            }
+            document.querySelector('.regency-top').style = cssRec
```
Comment (romanesent) [inline addons/regency_shopsite/static/src/js/frontend/header/store.js:82]: you need to change a specific style value, for example document.querySelector('.regency-top').style.background

File: addons/regency_shopsite/models/website.py [hunk 4]
```diff
@@ -18,10 +18,15 @@ def _header_preloaded_data(self):
...
-            'hotels': self.env.user.hotel_ids.read(['id', 'name']),
+            'hotels': self.env.user.hotel_ids.mapped(
+                lambda h: {field: h[field] for field in ['id', 'name', 'logo_url', 'background_url']}
+            ),
         }
         return data
 
+    def get_active_hotel_background_url(self):
+        self.env.user._active_hotel_id()
+
```
Comment (romanesent) [inline addons/regency_shopsite/models/website.py:29]: remove

**Pull request merged**